apps/web/src/app/error.tsx

```
@@ -0,0 +1,51 @@
+'use client';
+
+import React, { useEffect } from 'react';
+import { Button } from '@hypha-platform/ui';
+import { ReloadIcon } from '@radix-ui/react-icons';
+import { LightRootLayout } from '@web/components/light-root-layout';
+
+export default function Error({
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)